pkg/controller/telemetry/metric.go

```diff
@@ -508,7 +506,7 @@ func (m *MetricController) Run(ctx context.Context, mapOfTcpInfo *ebpf.Map) {
 			}
 
 			connectionLabels := connectionMetricLabels{}
-			if m.EnableConnectionMetric.Load() && reqMetric.duration > LONG_CONN_METRIC_THRESHOLD {
+			if m.EnableConnectionMetric.Load() {
```

Comment:
Shouldn't we only generate connection metrics for connection duration > 5s?
Otherwise how do you filter out the connection establishment report

Reply:
@yp969803